### PR TITLE
Major Revisons for the Previous Work Section

### PR DESCRIPTION
**Pull request by danich1**
Since this paper needed to be revised, I took the opportunity to restructure the entire previous work section. Instead of grouping approaches by edge type, I grouped works by approach type. Gets rid of duplicate citations and should be easier to read. Afraid it might still be dense, but will see once the whole paper has been merged.

### CODE REVIEW

**Review by dhimmel — APPROVED** [review]

File: content/02.introduction.md [hunk 1]
```diff
@@ -26,67 +26,77 @@ The re-use of label functions could dramatically reduce the number required to g
...
+#### Rule Based Extractors
+
+Rule based extractors rely heavily on expert knowledge to perform extraction.
+Typically, these systems use linguistic rules and heuristics to identify key sentences or phrases.
+For example, a hypothetical extractor focused on protein phosphorylation events would identify sentences containing the phrase "gene X phosphorylates gene Y" [@doi:10.1109/TCBB.2014.2372765].
+This word is a straightforward indication that two genes have a fundamental role in protein phosphorylation.
+Other phrase extractors have been used to identify drug-disease treatments [@tag:ctd_medline], pharmcogenomic events [@tag:pharmpresso] and protein-protein interactions [@tag:ppinterfinder; @tag:hpiminer].
+These extractors provide a simple but effective way to extract sentences; however, they depend on extensive knowledge about the text to be properly constructed.
+
+Besides using key phrases, a sentence's grammatical structure can be utilized to preform relationship extraction.
```
Comment (cgreene) [inline content/02.introduction.md]:
```suggestion
A sentence's grammatical structure can also support relationship extraction with dependency trees.
```

File: content/02.introduction.md [hunk 2]
```diff
@@ -26,67 +26,77 @@ The re-use of label functions could dramatically reduce the number required to g
...
+Extractors under this approach involve the use of dependency trees.
```
Comment (cgreene) [inline content/02.introduction.md]:
```suggestion
```

File: content/02.introduction.md [hunk 3]
```diff
@@ -26,67 +26,77 @@ The re-use of label functions could dramatically reduce the number required to g
...
+These trees are data structures that depict a sentence's grammatical relation structure in the form of nodes and edges.
+Nodes represent words and the edges represent the dependency type each word shares between one another.
+For example, a possible extractor would classify sentences as a positive if a sentence contained the following dependency tree path: "gene X (subject)-> promotes (verb)<- cell death (direct object) <- in (preposition) <-tumors (object of preposition)" [@tag:pkde4j].
+This approach provide extremely precise results, but the quantity of positive results remains modest as sentences appear in distinct forms and structure.
+Because of this limitation, recent approaches have incorporated methods on top of rule based extractors such as co-occurrence and machine learning systems [@doi:10.1186/s12859-018-2103-8; @tag:limtox].
+We discuss the pros and cons of added methods in a later section.
+For this project, we took a blind approach to construct our label functions; however, the discussed approaches in this section provides substantial inspiration for novel label functions in future endeavors.
```
Comment (cgreene) [inline content/02.introduction.md]: What does blind approach mean here?

File: content/02.introduction.md [hunk 4]
```diff
@@ -26,67 +26,77 @@ The re-use of label functions could dramatically reduce the number required to g
...
+Notable approaches exploit the fact that two entities can occur together in text.
+This event is referred to as co-occurrence.
+Extractors utilize these events in by generating statistics on the frequency of entity pairs occurring in text.
+For example, a possible extractor would say gene X is associated with disease Y, because gene X and disease Y appear together more often than individually [@tag:diseases].
+This approach has been used to establish the following relationship types: disease-gene relationships [@tag:diseases; @tag:polysearch; @tag:dg_text_pubmed; @tag:lgscore; @tag:full_text_co_abstracts; @tag:copub_discovery], protein-protein interactions [@tag:protein_protein_co_network; @doi:10.1093/database/bau012; @tag:full_text_co_abstracts], drug-disease treatments [@tag:abc_drugs], and tissue-gene relations [@doi:10.7717/peerj.1054].
+Extractors using the co-occurrence strategy provide exceptional recall results; however, these methods may fail to detect underreported relationships, because they depend on entity-pair frequency for detection.
+Junge et al. created a hybrid approach to account for this issue  [@tag:cocoscore].
```
Comment (cgreene) [inline content/02.introduction.md]:
```suggestion
Junge et al. created a hybrid approach to account for this issue using distant supervision to train a classifier to learn the context of each sentence [@tag:cocoscore].
```

File: content/02.introduction.md [hunk 5]
```diff
@@ -26,67 +26,77 @@ The re-use of label functions could dramatically reduce the number required to g
...
+Their approach utilized distant supervision to train a classifier to learn the context of each sentence.
```
Comment (cgreene) [inline content/02.introduction.md]:
```suggestion
```

File: content/02.introduction.md [hunk 6]
```diff
@@ -26,67 +26,77 @@ The re-use of label functions could dramatically reduce the number required to g
...
+Once training had been completed, they used the classifier to score every sentence within their corpus.
```
Comment (cgreene) [inline content/02.introduction.md]:
```suggestion
Once the classifier was trained, they scored every sentence within their corpus.
```

File: content/02.introduction.md [hunk 7]
```diff
@@ -26,67 +26,77 @@ The re-use of label functions could dramatically reduce the number required to g
...
+Each sentence's score was incorporated into calculating co-occurrence frequencies to establish relationship existence [@tag:cocoscore].
+Co-occurrence approaches are powerful in establishing edges on the global scale; however, they cannot identify individual sentences without the need for supervised methods.  
+
+Clustering is an unsupervised approach that extracts relationships from text by group similar sentences together.
```
Comment (cgreene) [inline content/02.introduction.md:68]:
```suggestion
Clustering is an unsupervised approach that extracts relationships from text by grouping similar sentences together.
```

File: content/02.introduction.md [hunk 8]
```diff
@@ -26,67 +26,77 @@ The re-use of label functions could dramatically reduce the number required to g
...
+A notable extractor used this technique to group sentences based on their grammatical structure [@tag:global_network].
```
Comment (cgreene) [inline content/02.introduction.md]: What is "a notable extractor" here? Maybe rephrase.

**Review by cgreene — APPROVED** [review]
Wow! This restructuring really helps! I have a few questions but I am happy with you clarifying those points and merging when you are ready.

File: content/02.introduction.md (hunk 3 above)
Comment (danich1) [reply content/02.introduction.md]: Ah I meant "blind approach" as in I constructed label functions without using any rules mentioned in discussed approaches.

Comment (danich1) [reply content/02.introduction.md]: Changing for clarity

**Pull request merged**